### ISSUE TITLE
Node FamilyInstance.ByPointAndLevel retains links to placed elements indefinitely

### ISSUE THREAD

**Issue by EmiXikes** [DynamoDS/DynamoRevit#1352]
## Dynamo version

Dynamo Revit 1.2.0.2690
## Revit version

Revit 2016
## Operating system

Windows 10
## What did you do?

Placed an element by using FamilyInstance.ByPointAndLevel node. 
Saved the dyn script. Closed it. 
Then reopened the script and tried to place another element at different coordinates.
## What did you expect to see?

A new element placed at the new coordinates.
## What did you see instead?

No new element was placed, instead the previously placed element was moved around.

![mpesfkd](https://cloud.githubusercontent.com/assets/22904432/19466272/86acc356-9512-11e6-8b77-a243a0382f47.png)

**Comment by mjkkirschner** [comment]
This is as designed currently.

What you are looking for is a way to break element binding between an element and a node. @kronz .

**Comment by EmiXikes** [comment]
I disagree. The node description says: "Place a Revit FamilyInstance given the FamilyType, it's coordinates in world space, and the Level". It says nothing about retaining any links to previously placed elements.
Note that this link remains even after closing and reopening Dynamo and Revit applications.
Do you mean to say, I am not allowed to reuse the said graph to place other elements?

**Comment by mjkkirschner** [comment]
Hi emixikes, yes that's exactly what I mean. Dynamo attempts to bind the node which constructed an element with that element so when the element is modified - dynamo will re execute that node keeping dynamo and rebut in sync.   This is a long requested addition to the current design - where it would be good in some cases to break that binding. The fact that it works on file open is part of that design 🙂

**Comment by EmiXikes** [comment]
Ok. Thank you for the explanation!
I got confused as there is nothing mentioned in the node description about such extensive binding effect.
Anyways, I'll be waiting for the feature that allows to break the link.

**Comment by ksobon** [comment]
@EmiXikes can you close this issue?

**Comment by jontziv** [comment]
Hello guys! I have the same issue with a hanger family... Is that resolved? How can I go about it? Revit 2015 Dyn veresion 1.2

**Comment by DavePlumb** [comment]
I just discovered this thread (and that behaviour) now - six months after this thread.
Is there anymore documentation/discussion on what @mjkkirschner says is "a long requested addition"?
I guess I don't understand how and where this would be useful.
And, I'd REALLY like to be able to use FamilyInstance.ByPoint more than once.
In my case, I'm using it to lay out seating along a path and as is, I can't use it to explore options.

**Comment by Kulkul7536** [comment]
Hello Everyone! I have posted solution here https://forum.dynamobim.com/t/familyinstance-bypoint-overwrites/10258/12?u=kulkul

Hope it Helps!

**Comment by DavePlumb** [comment]
KulKul;
One thing I just noticed in your node
It placed the families correctly on the Level I specified, but out FIRST LEVEL is set to 100'-0", and the new instances have an Offset of 100'-0" above FIRST LEVEL

**Comment by Thurops** [comment]
I can understand the desire for a link, but there are also a lot of situations where you don't want the link between the placement node and the placed families. Today I've been struggling with this same issue. The solutions that I found weren't helping me, so instead of solving it with Dynamo, I adjusted my workflow. Just select placed elements, copy to clipboard, delete placed elements and paste to same place.

A toggle with a boolean value to break the bond would be very nice.
